frontend/src/pages/projects/screens/projects/EmptyProjects.tsx

```diff
@@ -27,6 +27,7 @@ const EmptyProjects: React.FC = () => {
       <EmptyStateSecondaryActions>
         <Button
           href="/notebookController"
+          component="a"
           variant="link"
           onClick={(e) => {
             e.preventDefault();
```

Comment:
I wonder if this should not be more like [the companion button](https://github.com/opendatahub-io/odh-dashboard/blob/d02ba0bd62a26ada6fe2ae12f94d3593dbe6c6f0/frontend/src/pages/projects/screens/projects/ProjectListView.tsx#L95-L98) on the state-filled page.

Reply:
Great point - updated in https://github.com/opendatahub-io/odh-dashboard/pull/994/commits/26c978e27e3bbcae56e254a0180567126c06165c#diff-047fef1a71926288a0e0ce5836ce60d08cdb3656469356b77cb1c2609ef958cd